TCMSApp/src/client/app/tests/test-create.html

```diff
@@ -127,55 +127,38 @@ <h1 class="text-center">Create New Test Case</h1>
 
 <!-- STEPS -->
         <div class="well well-sm">
-          <div class="clearfix">
-            <div class="col-sm-6">
-              <div class="form-group">
-                <label for="step1">Step 1:</label>
-                <textarea class="form-control" rows="5"  class="form-control" id="step1" name="step1" placeholder="Step"></textarea>
+            <div  ng-repeat="s in vmCase.stepList track by $index">
+              <div class="clearfix">
+                <div class="col-sm-6">
+                  <div class="form-group">
+                    <label for="step{{s}}">Step: {{ $index + 1 }}</label>
+                    <textarea class="form-control" rows="5"  class="form-control" id="step{{s}}" name="step{{s}}" placeholder="Step"></textarea>
+                  </div>
+                </div>
+                <div class="col-sm-6">
+                  <div class="form-group">
+                    <label for="step{{s}}ExpRes">Expected result:</label>
+                    <textarea class="form-control" rows="5"  class="form-control" id="step{{s}}ExpRes" name="step{{s}}ExpRes" placeholder="Expected result"></textarea>
+                  </div>
+                </div>
               </div>
-            </div>
-            <div class="col-sm-6">
-              <div class="form-group">
-                <label for="step1ExpRes">Expected result:</label>
-                <textarea class="form-control" rows="5"  class="form-control" id="step1ExpRes" name="step1ExpRes" placeholder="Expected result"></textarea>
+              <div class="clearfix">
+                <div class="col-sm-12">
+                  <input type="file">
```

Comment:
I'd like to see the same icon as fa-trash here. for instance fa-upload or fa-paperclip